voltdbclient/node_conn.go

```diff
@@ -244,6 +244,23 @@ func (nc *nodeConn) loop(writer io.Writer, piCh <-chan *procedureInvocation, res
 			queuedBytes -= req.numBytes
 			delete(requests, handle)
 			if req.isSync() {
+
+				//WARNING: Please do not delete the below commented code it is used to
+				//generate the test samples for deserialization that are found in
+				//test_resources/deserialize
+				//
+				// TODO : Find a better way to generate this?
+				// file := os.Getenv("DES_BATCH")
+				// err = ioutil.WriteFile(file, resp.Bytes(), 0600)
+				// if err != nil {
+				// 	log.Fatal(err)
+				// }
+				// dir := filepath.Dir(file)
+				// err = ioutil.WriteFile(filepath.Join(dir, "handle"),
+				// 	[]byte(fmt.Sprint(handle)), 0600)
+				// if err != nil {
+				// 	log.Fatal(err)
+				// }
```

Comment:
The API makes it hard to access the raw response  bytes. This is the only way I can access them at the moment.

Also, this is insignificant. The samples once created they don't have to be recreated again. I left them just in case I need to recreate the samples for some reasons I don't know yet.